OrbitVersion/OrbitVersion.cpp.in

```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-#include "OrbitBase/OrbitVersion.h"
+#include "OrbitVersion.h"
```

Comment:
The header file should probably go to include/ dir - it is public

Reply:
done